app/controllers/TaskController.scala

```diff
@@ -99,13 +99,15 @@ class TaskController @Inject() (val messagesApi: MessagesApi)
   }
 
   private def buildFullParams(nmlParams: NmlTaskParameters, tracing: SkeletonTracing, fileName: String, description: Option[String]) = {
+    val parsedTracingBoundingBox = tracing.boundingBox.map(b => BoundingBox(b.topLeft, b.width, b.height, b.depth))
+    val bbox = if(nmlParams.boundingBox.isDefined) nmlParams.boundingBox else parsedTracingBoundingBox
```

Comment:
why does it make sense to give the nml bounding box precedence?

Reply:
`nmlParms`are the form-parameter that you specify when you are uploading an nml. There you can specify a bounding box explicitly (this is optional).
The bounding box `parsedTracingBoundingBox` is the bounding box which might be stored inside the nml (this is optional as well).

When you are uploading an nml and you are specifying the bounding box in the form-value explicitly then you are aware which bounding box you want. (Therefore this value is used)
When you don't specify a bounding box in the form-value then the bounding box of the parsed nml is used. (In case none of those two are specified, then the bounding box is `None`)

Reply:
@rschwanhold: Thanks for the explanation! I thought the variables are exactly the opposite^^
Therefore I renamed them [here](https://github.com/scalableminds/webknossos/pull/2827/commits/a4aecb2b44ccdc0c5a36a2d41b06348f4d83d2ab) to make this more clear.